src/generator/protocol/operations.ts

```diff
@@ -219,7 +219,7 @@ function createProtocolResponse(client: string, op: Operation): string {
   text += '\t}\n';
 
   const resp = op.responses![0];
-  const respObj = `${resp.language.go!.name}{StatusCode: resp.StatusCode}`;
+  const respObj = `${resp.language.go!.name}{RawResponse: resp.Response}`;
```

Comment:
Just double checking since most fields will be pointer to type this doesnt need to be &resp.Response?

Reply:
The `http.Response` wrapped in `azcore.Response` is already a pointer (same goes for requests).

Reply:
Ah you're right, thanks for the info!